lib/react_on_rails/helper.rb

```diff
@@ -327,7 +327,8 @@ def load_pack_for_component(component_name)
       end
 
       ReactOnRails::WebpackerUtils.raise_nested_entries_disabled unless ReactOnRails::WebpackerUtils.nested_entries?
-
+      puts "ZYXXYZ" if @self.nil?
```

Comment:
why `@self` and not `self`?

Reply:
Because I only want line 330~331 to run the debug statement once.